monitor/app/quicknode.go

```diff
@@ -0,0 +1,70 @@
+package monitor
+
+import (
+	"context"
+	"time"
+
+	"github.com/omni-network/omni/lib/errors"
+	"github.com/omni-network/omni/lib/ethclient"
+	"github.com/omni-network/omni/lib/log"
+	"github.com/omni-network/omni/lib/netconf"
+)
+
+const (
+	quicknodePublicRPC = "https://damp-wandering-gadget.omni-omega.quiknode.pro"
```

Comment:
actually, lets rather make this a flag/config item. Then we can specify different values in ops repo (per network) and not leak this to public